documentation/documentation/admin/installing-plv8-windows.md

```diff
@@ -52,6 +56,22 @@ The `Program Files` and `9.5` folders should correspond to the bit and version o
 
 > C:\Program Files (x86)\PostgreSQL\9.6\
 
+### Distributions from Xtuple
+
+The zip contains the folders for: 
```

Comment:
Rather than listing the folders for each PG version, remove those entries and simplify it as below:
> Run the batch file `install_plv8.bat` available in the relevant Postgres version folder from a command window running in administrative mode and provide the path for your Postgres installation.